README.md

```diff
@@ -51,9 +33,24 @@ roseus hrp2jsknt-interface.l
 
 ### Package Description
 
-hrpsys_ros_bridge_tutorialsls
+### [hrpsys_ros_bridge_tutorials]
 --------------
-This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common). 
+This is examples and tutorials for [rtmros_common/hrpsys_ros_bridge](https://github.com/start-jsk/rtmros_common).
 
 Euslisp tutorial wiki is written in [ROS wiki](http://wiki.ros.org/rtmros_common/Tutorials/WorkingWithEusLisp).
 
+### [hrpsys_gazebo_tutorials]
+
+This package is a collection of examples for using hrpsys_gazebo system and utility scripts.
+
+- You should prepare robot model file. Supported types of model file are collada(openrave) and VRML(openhrp3). URDF and OpenRAVE xml can be used by converting to collada.
```

Comment:
Is this section additional? In my understanding, this package can be used with SampleRobot in default, and that procedure doesn't require explicit preparation of robot model by user.
Also, you should first say where to put model file and how to use it in gazebo simulation.
After that, you can explain about details (what is automatically generated, etc.)
In my understanding, when `rtmros_hrp2` is downloaded, `catkin build` is only needed to use HRP2, isn't it?
So I think you should explain as follows:
1. Explain SampleRobot (only `catkin build` is needed)
2. Explain HRP2 (downloading `rtmros_hrp2` and `catkin build` are needed)
3. Explain user-defined robot (placing model file to right place, running script, and `catkin build` are needed)
4. Explain about details (what is automatically generated, etc.)

Reply:
I understand this is copied from `rtmros_gazebo`, but it's hard to understand...

Reply:
I understand this section is for `hrpsys_ros_bridge_tutorials`. Now my comment seems meaningless